### ISSUE TITLE
Can I know how the loss in SSD should look like?

### ISSUE THREAD

**Issue by yxchng** [amdegroot/ssd.pytorch#48]
Do you still keep your loss info? Currently my loss keep hovers at 20. Can I know what loss value is considered reasonable?

**Comment by DuinoDu** [comment]
My loss ends at around 6 and mAP (trained on voc2007trainval and test on voc2007test) is 0.683, using SSD300

**Comment by 201814476** [comment]
速度还可以，准确率有点低

**Comment by jaychou790118** [comment]
你好，你有没有用demo测试框




| |
13032926926
|
|
邮箱13032926926@163.com
|

Signature is customized by Netease Mail Master

On 08/04/2020 16:14, 201814476 wrote:

速度还可以，准确率有点低

**Comment by 201814476** [comment]
用demo测试框报错AttributeError: 'NoneType' object has no attribute 'shape'

**Comment by smallsun1122** [comment]
您好，请问一下，您这边在训练的过程中利用visdom显示的loss可以正常显示吗？我这边利用visdom显示loss的时候刚刚开始是有数据的，但是过一会数据就不会变化，然后刷新一下浏览器，图就没有了。您遇到过这种情况吗？
